### PR TITLE
Added questions with priority 30 and 31

### PR DESCRIPTION
**Pull request by khusharth**
Hey @tanaypratap i have added the two new FAQ's. Please review them.

### CODE REVIEW

File: content/faq/31_what-do-i-do-if-i-get-error-while-coding/index.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: what do i do if i get error while coding?
```
Comment (tanaypratap) [inline content/faq/31_what-do-i-do-if-i-get-error-while-coding/index.md]: this should be:
`what do I do if I get an error in a public library which I am using?`

File: content/faq/30_what-should-i-expect-from-mentors/index.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
```
Comment (tanaypratap) [inline content/faq/30_what-should-i-expect-from-mentors/index.md]: I think we now have a more detailed answer to this. It includes how you should raise issue in this repo to ask for help.

We should make this answer more rich. Talk to Sahithi from the volunteers team for this.

Comment (khusharth) [reply content/faq/30_what-should-i-expect-from-mentors/index.md]: In my latest FAQ PR i have added a new question (47) -> "_how to make sure that you get a mentor for a review?_" 
Will that question replace this one?
It has a really good and detailed answer.

Comment (tanaypratap) [reply content/faq/30_what-should-i-expect-from-mentors/index.md]: yes.. that is great.. remove this question from here then..

**Comment by tanaypratap** [comment]
https://2020.teamtanay.jobchallenge.dev/98_where_can_i_clear_my_doubts/

Also, remove/edit this question. Let's start sending all traffic to Github issues. Easier to track and threaded conversation.

File: content/faq/31_what-do-i-do-if-i-get-error-while-coding/index.md (hunk 1 above)
Comment (khusharth) [reply content/faq/31_what-do-i-do-if-i-get-error-while-coding/index.md]: Done.

**Pull request merged**